protocols/bgp/server/fsm_idle.go

```diff
@@ -16,7 +16,7 @@ func newIdleState(fsm *FSM) *idleState {
 }
 
 func (s idleState) run() (state, string) {
-	if s.fsm.peer.reconnectInterval != 0 {
+	if !s.fsm.peer.passive {
```

Comment:
Hm, I'm wondering if we need to keep the check for `!= 0` here, too. @taktv6 any thoughts?

Reply:
I think this is a good suggestion, thanks.